frog/imports/api/templates.js

```diff
@@ -0,0 +1,61 @@
+import { Meteor } from 'meteor/meteor';
+import { Mongo } from 'meteor/mongo';
+import { uuid } from '/imports/frog-utils';
+import { Graphs } from './graphs';
+
+export const Templates = new Mongo.Collection('templates');
+
+export const addTemplate = (name, graph) => {
+  const templateId = uuid();
+  Templates.insert({
+    name: name,
+    graph: graph,
+    _id: templateId,
+    ownerId: Meteor.userId(),
+    createdAt: new Date()
+  });
+};
+
+export const findTemplate = id => {
```

Comment:
I had planned to do more with this function. Just in case we need to make any changes in all the graphs created using the template, so I was planning to fetch the graphs and add a graphs object containing ids. I don't think we need this as of now though, so I'll minify this!